docs/css/theme.css

```diff
@@ -1,5 +1,6 @@
 /* Theme colors */
-a {
+a,
+h1 {
   color: #00A3E6;
```

Comment:
`#00A3E6` against `#FFFFFF` background is probably hard for some users to read because of the low contrast ratio. Could we consider some options for more accessible theme colors?

Reply:
Thanks for the feedback. Personally the color contrast is not a problem for me, but i understand some people might have issue. The blue should work with the color palette which is designed to be vibrant/energetic. I updated to the color to Material Blue 500 #2196f3, which should work well with others, and improve contrast. 

<img width="884" alt="screen shot 2018-12-13 at 11 28 15 am" src="https://user-images.githubusercontent.com/406636/49962872-4ee13b80-fecb-11e8-8fb6-4138ea2dfddb.png">

<img width="907" alt="screen shot 2018-12-13 at 11 29 02 am" src="https://user-images.githubusercontent.com/406636/49962881-543e8600-fecb-11e8-9acb-d48b12dcf512.png">